lib/api/get/meetings.ts

```diff
@@ -1,7 +1,9 @@
 import { RRule, RRuleSet } from 'rrule';
 import { nanoid } from 'nanoid';
 
-import { Meeting, MeetingsQuery, Timeslot } from 'lib/model';
+import { Meeting } from 'lib/model/meeting';
```

Comment:
Make sure that you're following our ESLint import sorting rules. I'm pretty sure that these imports (because they're only importing a single item each) should go after the imports with multiple items.

You should make sure that whatever IDE you're using is setup with ESLint is is following our configuration.